app/lib/create-reference.js

```diff
@@ -0,0 +1,11 @@
+const { v4: uuid } = require('uuid')
+
+/**
+ * Generate unique reference number
+ * ex. VV22-B471-F25C
+ * @returns string
+ */
+module.exports = () => {
+  const id = uuid().split('-').shift().toLocaleUpperCase('en-GB').match(/.{1,4}/g).join('-')
+  return `VV${new Date().getFullYear().toString().slice(-2)}-${id}`
```

Comment:
I've been thinking about this format, specifically the year element. Given it is only us who care about it I think we should make it simpler and remove the year element so it becomes `VV-XXXX-XXXX`. The main reason for this it because I had originally thought the year would represent the financial year as I figured that would have been useful but as it doesn't matter removing the year altogether makes most sense.

Reply:
ok